packages/admin/src/DirtyHandlerApiContext.tsx

```diff
@@ -1,16 +1,18 @@
 import * as React from "react";
 
+import { SubmitResult } from "./form/SubmitResult";
+
 export interface IDirtyHandlerApiBinding {
     isDirty: () => boolean;
-    submit: () => Promise<void>;
+    submit: () => Promise<SubmitResult | void>;
     reset: () => void;
 }
 export interface IDirtyHandlerApi {
     registerBinding: (cmp: object, binding: IDirtyHandlerApiBinding) => void;
     unregisterBinding: (cmp: object) => void;
     isBindingDirty: () => Promise<boolean>;
     resetBindings: () => Promise<void>;
-    submitBindings: () => Promise<void>;
+    submitBindings: () => Promise<Array<SubmitResult | undefined>>;
```

Comment:
please convert that alwasys into a SumitResult, so the return value is Array<SubmitResult>

Reply:
Then it is not backwards compatible again. e.g. if someone relies on submitBindings returning `[undefined]`.

(Inside comet-admin `submitBindings` is only used in the `EditDialog`. There it wouldn't be a problem. But I don't know if it is used somewhere in another project)

Reply:
submitBindings did not have (before this PR) a return value, so adding one is no problem.

Reply:
Okay